src/main/java/core/basesyntax/BalanceOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the empty line at the start of the class definition.